prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics-release.yaml

```diff
@@ -313,7 +313,29 @@ periodics:
       - 'GKE_RELEASE_CHANNEL=regular'
       - 'E2E_CLUSTER_PREFIX=standard-regular-gcenode'
       - 'E2E_NUM_CLUSTERS=1'
-      - 'E2E_ARGS=--gcenode -run=TestGCENode'
+      - 'E2E_GIT_PROVIDER=csr'
```

Comment:
AFAIK we're not ready to port these new configs to the release CI, I think this should be done after 1.18 is cut

Reply:
Ah, right. Thanks for catching this. Reverted.